sdk/lib/web3.js

```diff
@@ -68,7 +68,13 @@ async function getLogs({ web3, scan, limiter, batchSize, target, topic, topics,
   // assume scattered events can be returned in one page
   // TODO we may need to avoid this assumption
   // and add pagination support at some point
-  const txs = await scan.getTxList({ address: target, startBlock: fromBlock, endBlock: toBlock });
+  const [externalTxs, internalTxs] = await Promise.all(
+    [
+      await scan.getTxList({ address: target, startBlock: fromBlock, endBlock: toBlock }),
+      await scan.getTxListInternal({ address: target, startBlock: fromBlock, endBlock: toBlock })
```

Comment:
this line is quite problematic in real world.

for pancakeswap factory https://bscscan.com/txsInternal?a=0xbcfccbde45ce874adcb698cc183debcf17952812, there're more than 100k internal tx available, which will take endless time to pull.
one thing I noticed is that in those 100k internal tx, most of them are not related to pair creation.

we may have to figure out a better way to filtering internal txs.